### PR TITLE
Devise Recipe

### PR DESCRIPTION
**Pull request by guilleiguaran**

### CODE REVIEW

**Comment by guilleiguaran** [comment]
Updated, ruby_parser is required only for Haml views according to https://github.com/plataformatec/devise/commit/3fd206c5916361aabf941c4432ef2518220ca678

File: lib/template_framework/template_runner.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ $LOAD_PATH.unshift(File.expand_path('./../../', __FILE__))
 require 'rails_templater'
 require File.join(File.dirname(__FILE__), 'core_ext.rb')
 
-%w(default orm testing_framework javascript_framework template_engine sass cucumber).each do |recipe|
+%w(default orm testing_framework javascript_framework template_engine sass cucumber devise).each do |recipe|
```
Comment (guilleiguaran) [inline lib/template_framework/template_runner.rb:6]: Maybe a good idea create a new recipe called "misc" and put here the questions about other optional recipes that aren't part of a basic Rails stack as Devise and SimpleForm. This recipe then can apply devise.rb, simple_form.rb, etc

**Comment by kfaustino** [comment]
I've put the recipe into a branch. I will merge it in later this week.

**Comment by brendte** [comment]
good addition!

**Pull request closed without merging**